resources/views/components/link-arrow.blade.php

```diff
@@ -0,0 +1,9 @@
+<a
+    href="{{ $link['url'] }}"
+    class="group block leading-loose flex {{ Request::is($link['url']) ? 'text-blue' : '' }}"
```

Comment:
This check for active link is something I have seen in the webumenia repo, but it doesn't seem to work for the `/test` slugs. Any suggestions here?

Reply:
well... no easy answer here 😄 
but I think to make that component as universal, as possible... let's go for something like 
```suggestion
    class="group block leading-loose flex {{ (url()->current() == $link['url']) ? 'text-blue' : '' }}"
```
just to explain. it doesn't work, because the request is `"test"` but the `$link['url']` is `"/test"`
anyway.... it's fine for now with this dummy data... and I am happy to take care for that with filling with real data :)